tests/test_utils/test_transactiontestcase.py

```diff
@@ -26,3 +46,22 @@ def test(self, call_command):
             reset_sequences=False, inhibit_post_migrate=True,
             database='default', verbosity=0,
         )
+
+
+class TestDataConsistencyOnTearDown(TestSerializedContentMockMixin, TransactionTestCase):
+    """
+    TransactionTestCase._fixture_teardown() is flushing the whole database.
+    If an other test after this one needs some data coming from initial data migrations,
+    there data won't be found.
+    One solution is to load again initial data migrations after the flush called on _fixture_teardown().
+    """
+    available_apps = ['test_utils']
+    serialized_rollback = True
+    initial_data_migration = '[{"model": "test_utils.car", "pk": 1, "fields": {"name": "K 2000"}}]'
+
+    def _post_teardown(self):
```

Comment:
These tests won't show the problem, they will be ok because the current logic is loading data from migrations **before** launching each test.
But after these 2 tests, your database will be empty, which is the current problem with `--keepdb` option.

I overwrite `_post_teardown()` to be able to test that my data are still there **after** each test, and `tearDown()` is not enough because the cleaning logic is in `_post_teardown()`.